.github/workflows/terraform_build.yaml

```diff
@@ -12,14 +12,15 @@ jobs:
 
     - name: Test with pytest
       run: |
-        ./scripts/test_v2
         ./scripts/test
+        ./scripts/test_v2 --no_build
```

Comment:
Curious what impact this update will have on the tests and updating dependencies. Was it needed for the branch fixes you made?